exchanges/bybit/bybit_wrapper.go

```diff
@@ -312,7 +314,7 @@ func (by *Bybit) FetchTradablePairs(ctx context.Context, a asset.Item) (currency
 	default:
```

Comment:
So to further talk about this. What I'm after here is for the `asset.Options` asset type, you will need to make two API requests to get all tradable pairs because Bybit supports BTC and ETH options. By default, the request for options will default to `BTC` if the base currency is empty.

Because of this you'll need to add a new `case` for `asset.Options` where you make an additional API request with the base currency set to ETH and then add those to tradable pairs for options.

I hope that makes sense. It looks like you've done this for the `UpdateTickers` function though, so thank you!